modules/nw-customize-ingress-error-pages.adoc

```diff
@@ -0,0 +1,104 @@
+// Module filename: nw-customize-ingress-error-pages.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-customize-ingress-error-pages_{context}"]
+= Customizing HAProxy error code response pages
+
+As a cluster administrator, you can specify a custom error code response page for either 503, 404, or both error pages. The HAProxy router serves a 503 error page when the application pod is not running or a 404 error page when the requested URL does not exist. For example, if you customize the 503 error code response page, then the page is served when the application pod is not running, and the default 404 error code HTTP response page is served by the HAProxy router for an incorrect route or a non-existing route.
+
+Custom error code response pages are specified in a config map then patched to the Ingress Controller. The config map keys have two available file names as follows:
+`error-page-503.http` and `error-page-404.http`.
+
+Custom http error code response pages must follow the link:https://www.haproxy.com/documentation/hapee/latest/configuration/config-sections/http-errors/[HAProxy HTTP error page configuration guidelines]. Here is an example of the default {product-title} HAProxy router link:https://raw.githubusercontent.com/openshift/router/master/images/router/haproxy/conf/error-page-503.http[http 503 error code response page]. You can use the default content as a template for creating your own custom page.
```

Comment:
Please upcase HTTP, as in "Custom HTTP error..."

If you keep the link to the GitHub repo, then please have Eric Rich indicate approval.  See the [links to external websites](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites) section of the doc guidelines.
> You cannot link to a repository that is hosted on www.github.com. 

If you can get that rule changed, even better.

Reply:
@sferich888 can you approve this GitHub link?

Reply:
+1 you can use github links for this.